coalahtml/_coalahtml/app/controllers/logs.js

```diff
@@ -9,5 +9,5 @@ angular.module('coalaHtmlApp')
         $scope.data = response.logs;
       });
     };
-    parseCoalaProject("tests/test_projects/simple");
+    parseCoalaProject("data");
```

Comment:
Hmmm, so in the new package structure, will the python code be generating the `*.json`s in this directory ? Shouldn't the path here be the path for the place the python code saves the *.json ?